src/modules/helpers/index.ts

```diff
@@ -629,7 +629,7 @@ export class HelpersModule {
    *
    * @example
    * enum Color { Red, Green, Blue }
-   * faker.helpers.enumValue(Color) // 'Green'
+   * faker.helpers.enumValue(Color) // 1
```

Comment:
while correct, this example doesn't seem very explanatory. (why 1?). It would be helpful to clarify this equals the value of one of the enum values, something like this?
```
const randomColor = faker.helpers.enumValue(Color) // randomColor is 1 
randomColor === Color.Green // true
```

Reply:
IMO we can achieve the same with the comment:

```suggestion
   * faker.helpers.enumValue(Color) // 1 (Green)
```